README.md

```diff
@@ -347,3 +347,25 @@ The `beaver dlang make` command also uses some utilities that could come handy
 if you want to write a custom build script. Take a look at the `lib/dlang.sh`
 file, you'll find the utility functions with documentation in there.
 
+
+### Codecov support
+
+Beaver includes a convenient wrapper to send [codecov.io](https://codecov.io/)
+reports. Just make sure the project was already built with coverage support and
+then use something like this in the `.travis.yml` file:
+
+```yml
+after_success: beaver dlang codecov [OPTIONS]
```

Comment:
If I understand correctly, the script auto-detects languages so it should be just plain `beaver codecov`.

Reply:
It has many option, like sending env vars, sending labels, enabling or disabling features, I can already see people wanting to use those.

Reply:
?

All I was saying that it is semantically `beaver codecov [OPTIONS]`, not `beaver dlang codecov [OPTIONS]` because it will work with any supported code cov language.

Reply:
But this script only works with D projects, it assuming a lot of stuff that work only with them (like that report files have the pattern `.*.lst`, and it passes the `F` and other Makd env vars).

We need something specific for D for now, so I prefer to limit to it to simplify things and get the most out of this setup.